ludicrousdb/includes/class-ludicrousdb.php

```diff
@@ -1708,6 +1728,33 @@ public function get_lag_cache() {
 		return $this->check_lag();
 	}
 
+
+	/**
+	 * @param $dbh
+	 *
+	 * @return bool
+	 */
+	function should_mysql_ping( $dbh ) {
```

Comment:
I assume is a public function, but, what if is a static function? We could improve the war we call that function.